src/containers/location/providers/LocationsContainer.js

```diff
@@ -124,9 +133,27 @@ const LocationsContainer = () => {
   }
 
   const hasSearched = !isStandby(searchLocationsRS);
-  const isLoading = isPending(searchLocationsRS);
+  const isLoading = isPending(geocodePlaceRS)
+    || isPending(searchLocationsRS)
+    || isPending(loadCurrentPositionRS);
```

Comment:
No change required. There is a util for this:

```
import { ReduxUtils } from 'lattice-utils';
const { reduceRequestStates } = ReduxUtils;
const isPending :boolean = reduceRequestStates([geocodePlaceRS, searchLocationsRS, loadCurrentPositionRS]);
```

Reply:
kool

Reply:
`reduceRequestStates` returns a `RequestState`, so this should be
```
```